### PR TITLE
change image markup

### PR DESCRIPTION
**Pull request by mayank1211**
## Description

Images - Add `size` and `srcset` attributes to the images component for responsive loading of images. ([Issue 422](https://github.com/nhsuk/nhsuk-frontend/issues/422))

Resolves https://github.com/nhsuk/nhsuk-frontend/issues/422

## Checklist

- [ ] Tested against our [testing policy](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/testing.md) (Resolution, Browser & Accessibility)
- [ ] Follows our [coding standards and style guide](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/coding-standards.md)
- [x] CHANGELOG entry

### CODE REVIEW

File: packages/components/images/README.md [hunk 1]
```diff
@@ -30,13 +31,28 @@ Find out more about the images component and when to use it in the [NHS digital
 
 {{ image({
   "src": "https://assets.nhs.uk/prod/images/S_1017_allergic-conjunctivitis_M15.2e16d0ba.fill-320x213.jpg",
+  "sizes": "(min-width: 1020px) 320px,
```
Comment (mikemonteith) [inline packages/components/images/README.md]: At desktop size, these images are displayed at 420px, not 320px.

**Comment by mikemonteith** [comment]
Probably needs a CHANGELOG entry. @AdamChrimes ?

**Comment by mikemonteith** [comment]
Looks like the backstop tests are failing. https://travis-ci.org/nhsuk/nhsuk-frontend/jobs/592032901

**Comment by AdamChrimes** [comment]
Thanks @mayank this looks good, I'll have a quick test.

Please could you resolve the conflict with the `CHANGELOG.md`? Give me a shout if you need help with conflicts.

**Comment by AdamChrimes** [comment]
Just need to check the indenting on the HTML and Nunjuck macro snippets in the `README`

We use soft tabs with two spaces and nested elements should be indented once (two spaces).

[Coding standards - Syntax](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/coding-standards.md#syntax)

Screenshot 2019-10-09 at 11 50 32

**Comment by mikemonteith** [comment]
Backstop tests are still failing on this branch.
https://travis-ci.org/nhsuk/nhsuk-frontend/jobs/595674785 4 failures.

File: app/components/pagination/index.njk [hunk 2]
```diff
@@ -12,7 +12,7 @@
         <div class="nhsuk-grid-column-two-thirds">
           {{ pagination({
             "previousUrl": "/section/treatments",
-            "previousPage": "Treatments",
+            "previousPage": "where confidential patient information is used",
```
Comment (AdamChrimes) [inline app/components/pagination/index.njk]: Seems to be a random change to the pagination component here. BackstopJS tests failing because of it.

**Comment by mikemonteith** [comment]
There are 12 commits here for one change. Would be good to rebase on master before merging.

**Comment by AdamChrimes** [comment]
I've undone the change to the pagination component so that BackstopJS tests are passing.

I've also tidied up some of the README adding the new arguments to the nunjucks macro table and cleaned up some of the examples.

Will squash all these commits into 1 on merge.

**Comment by AdamChrimes** [comment]
@mikemonteith you OK to give this another review please?

**Pull request merged**